manifests/config/global/securityroles.pp

```diff
@@ -0,0 +1,18 @@
+#
+# Author: Zoltan Lanyi <zoltan.lanyi@gmail.com>
+# Date  : 06.06.2016
+# 
+
+define rundeck::config::global::securityroles ($security_role = $title) {
+  augeas { "rundeck/web.xml/security-role/role-name/$security_role":
```

Comment:
i havent found any solution to handle augeas with puppet loop. Another idea is using template for web.xml instead of augeas.